BLSQ/iaso
# [IA-3631]  Replaced `assertEquals` by `assertEqual`

Replaced `assertEquals` by `assertEqual`

Related JIRA tickets : IA-3631

## Self proofreading checklist


## Doc
/

## Changes

/

## How to test

If you have python 3.12 or higher, run all tests. There won't be any error linked to `assertEquals`.
If your python is lower than 3.12, you won't see any change, all test will behave as usual.


## Print screen / video

The error is not linked to this change, I also had it with `assertEquals`
![image](https://github.com/user-attachments/assets/8eaefa07-c4ab-4a4b-b627-50e5f6c82f53)



## Notes

/
